lms/djangoapps/discussion/static/discussion/js/discussion_board_factory.js

```diff
@@ -61,7 +61,7 @@
 
                 // Set up a router to manage the page's history
                 router = new DiscussionRouter({
-                    courseId: options.courseId,
+                    rootUrl: options.root_url,
```

Comment:
Hmm.  So the `options` here in `options.root_url` is not the same as the `options` in `options.rootUrl` in `discussion_router.js` below.  

It does seem like Python should speak JS-ese when sending it data, but it sounds like that is a large refactor.  I guess I still need clarity on the different `options` to understand the level of confusion here.